python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -786,6 +792,82 @@ def maskSaturatedPixels(self, badAmpDict, ccdExposure, detector, detectorConfig,
                 badAmpDict[ampName] = True
                 maskView |= maskView.getPlaneBitMask("BAD")
 
+        # Edge bleed masking
+        if self.config.doITLEdgeBleedMask and detector.getPhysicalType() == 'ITL':
+
+            # Get footprint of saturated pixels
+            thresh = afwDetection.Threshold(limits[self.config.saturatedMaskName])
+            fpList = afwDetection.FootprintSet(maskedImage, thresh).getFootprints()
+
+            # Go through footprints to select saturated cores
+            for i in range(len(fpList)):
+                if fpList[i].getArea() > 10000.:
+
+                    # Get centroid of saturated core
+                    xCore, yCore = fpList[i].getCentroid()
+                    # Get the number of saturated columns around the centroid
+                    widthSat = numpy.sum(fpList[i].getSpans().asArray(), axis=1)[int(yCore)]
```

Comment:
This seems wasteful to do a whole summation and then take one value.  Why not `numpy.sum(fpList[i].getSpans().asArray()[:, int(yCore)]` (or I might have this transposed but you get the idea).